internal/checkers/negative_postive.go

```diff
@@ -0,0 +1,81 @@
+package checkers
+
+import (
+	"golang.org/x/tools/go/analysis"
+
+	"github.com/Antonboom/testifylint/internal/analysisutil"
+)
+
+// NegativePositive detects situations like
+//
+//	assert.Greater(t, 0, value)
```

Comment:
will be updated with new cases (see below) + README

Reply:
You only refer to these right ?

https://github.com/Antonboom/testifylint/pull/78#discussion_r1561432681